app/models/user.rb

```diff
@@ -11,12 +13,10 @@ def normalise_queue
   end
 
   def video_is_queued?(video)
```

Comment:
This method should be in refactored to `User` model, and probably renamed very slightly to something like 'has_queued?(video)`. Think about it.